app/code/community/SomethingDigital/PageCacheParams/Model/Processor.php

```diff
@@ -73,7 +73,7 @@ protected function processUri(&$uri)
                 $uri = substr($uri, 0, $pos);
             } else {
                 // Changed to a shorter query, reinsert after '?'.
-                $uri = substr($uri, 0, $pos + 1) . $query;
+                $uri = substr($uri, 0, $pos + 1).$query;
```

Comment:
This is part of the Symfony standards for some weird reason, but it's not part of PSR-2:
https://github.com/symfony/symfony/issues/3135
https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/783